tests/plotting/test_plotting.py

```diff
@@ -302,22 +302,21 @@ def test_pbr(sphere, verify_image_cache):
     pl.show()
 
 
-@pytest.mark.skip_windows
-@pytest.mark.skip_mac('MacOS CI fails when downloading examples')
-@pytest.mark.parametrize('resample', [True, False])
-@pytest.mark.needs_vtk_version(9, 1)  # GC issue
-def test_set_environment_texture_cubemap(sphere, verify_image_cache, resample):
+@pytest.mark.parametrize('resample', [True, 0.5])
+@pytest.mark.needs_vtk_version(9, 2)
+def test_set_environment_texture_cubemap(resample, verify_image_cache):
     """Test set_environment_texture with a cubemap."""
-    verify_image_cache.high_variance_test = True
-
-    texture = examples.download_sky_box_cube_map()
+    if resample:
+        # Skip due to large variance
+        verify_image_cache.windows_skip_image_cache = True
+        verify_image_cache.macos_skip_image_cache = True
```

Comment:
Since `resample` is always True, the `if` block can be removed.
```suggestion
    # Skip due to large variance
    verify_image_cache.windows_skip_image_cache = True
    verify_image_cache.macos_skip_image_cache = True
```